packages/react-native/sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -10,7 +10,8 @@ react_native_path = File.join(__dir__, "..", "..")
 
 # package.json
 package = JSON.parse(File.read(File.join(react_native_path, "package.json")))
-version = package['version']
+rn_version = package['version']
+version = findLastestVersionWithArtifact(rn_version) || rn_version
```

Comment:
Could we mark this diff with " # [macOS / # macOS] " tags as documented here? docs/DiffsWithUpstream.md

Reply:
Sure, updated